nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestPutSQL.java

```diff
@@ -338,14 +342,12 @@ public void testFailInMiddleWithBadStatementRollbackOnFailure() throws Initializ
         runner.enqueue("INSERT INTO PERSONS_AI (NAME, CODE) VALUES ('Tom', 3)".getBytes());
         runner.enqueue("INSERT INTO PERSONS_AI (NAME, CODE) VALUES ('Harry', 44)".getBytes());
 
-        try {
-            runner.run();
-            fail("ProcessException should be thrown");
-        } catch (AssertionError e) {
-            assertTrue(e.getCause() instanceof ProcessException);
-            runner.assertTransferCount(PutSQL.REL_FAILURE, 0);
-            runner.assertTransferCount(PutSQL.REL_SUCCESS, 0);
-        }
+        final AssertionError e = assertThrows(AssertionError.class, () -> {
+           runner.run();
+        });
+        assertTrue(e.getCause() instanceof ProcessException);
```

Comment:
This should be changed to use `assertInstanceOf`

Reply:
Making the changes in TestPutSQL